changelogs/fragments/alpine-libuser-community.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+- group integration tests - Update ``Alpine`` repo where ``libuser`` has moved to
```

Comment:
No changelog needed for changes only affecting integration tests.

Reply:
removed